numpy/typing/tests/data/reveal/fromnumeric.pyi

```diff
@@ -97,6 +97,11 @@ assert_type(np.argmin(AR_b, axis=0), Any)
 assert_type(np.argmin(AR_f4, axis=0), Any)
 assert_type(np.argmin(AR_f4, out=AR_subclass), NDArraySubclass)
 
+OUT_I = np.empty((), dtype=np.intp)
```

Comment:
There's some shape-typing involved here, which is causing the mypy tests to fail. It's probably easiest to just use `AR_i8` instead.